src/pymediainfo/__init__.py

```diff
@@ -1,7 +1,9 @@
-# vim: set fileencoding=utf-8 :
 """
 This module is a wrapper around the MediaInfo library.
 """
+
+from __future__ import annotations
```

Comment:
May I ask what parts of the typing make use of this?

Reply:
It's to avoid checking types at run type. Like that you can use type annotations from newer python versions and still be able to.run the code with say python 3.8.